source/changelog.txt

```diff
@@ -12,10 +12,29 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.0.2
+------
+
+*Released October 16, 2023*
+
+- Upgrades to Node.js version 20.8.1. Node 20.8.1 addresses security
+  issues. For more information, see
+  `Node.js security releases
+  <https://nodejs.org/en/blog/vulnerability/october-2023-security-releases>`__ and 
+  `CVE-2023-45133 <https://nvd.nist.gov/vuln/detail/CVE-2023-45133>`__.
+- `CVE-2023-45143 <https://nvd.nist.gov/vuln/detail/CVE-2023-45143>`__  
+  affects the ``fetch()`` API that is available in ``mongosh`` 2.x.
+
+.. important::
+    ``mongosh`` 1.x does not address the above security issues. Please upgrade to 
```

Comment:
Consider:
``mongosh`` 1.x does not resolve the security issues in the previous list. For improved security, please upgrade to 
    ``mongosh`` 2.0.2.